package.json

```diff
@@ -11,19 +11,20 @@
   "bugs": "https://github.com/ain/smartbanner.js/issues",
   "dependencies": {},
   "devDependencies": {
+    "add-banner": "ain/add-banner#121b56d",
```

Comment:
I can't see a change here, let's then keep the original file to avoid the diff of possible.